src/ui/components/Button/index.js

```diff
@@ -9,31 +9,32 @@ import './Button.scss';
 
 export default class Button extends React.Component {
   static propTypes = {
-    appearance: PropTypes.string,
```

Comment:
I agree. Asking each caller to know the class name puts an undue mental burden on them (they have to look it up in the css or try to find an example of a similar call). 

Also, with Flow annotations, an open ended string property like this is easier to deal with. For example, you can glance at the props type and know what to do:

```js
type ButtonProps = {|
  appearance: 'light' | 'dark',
  ...
|};

export class Button extends React.Component {
  props: ButtonProps;
  ...
}
```